.gitattributes

```
@@ -0,0 +1 @@
+* text eol=lf
```

Include a proper line ending.
```suggestion
* text eol=lf

```